### PR TITLE
fix: backpublish fixes

### PR DESCRIPTION
**Pull request by dos65**
Checked it on [v4.5.5 + 2.12.16](https://github.com/scalameta/scalameta/runs/6799302039?check_suite_focus=true)

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -66,7 +66,10 @@
 
 ## Releasing for a specific version
 
-Realeasing for a specific existing version, which is needed for example in case of new Scala version, is more complicated. The flow is as follows:
+Realeasing for a specific existing version might be needed for example in case of new Scala version.
```
Comment (kitbellew) [inline RELEASING.md:69]: releasing

File: RELEASING.md [hunk 2]
```diff
@@ -66,7 +66,10 @@
...
+
+To perfrom it on CI create and push a tag with a special name: `semanticdb_v${existing-scalameta-version}_${scala_version}`.
```
Comment (kitbellew) [inline RELEASING.md:71]: perform. also comma after CI.

File: RELEASING.md [hunk 3]
```diff
@@ -66,7 +66,10 @@
...
+You can also do it manually.The local flow is:
```
Comment (kitbellew) [inline RELEASING.md:72]: space after dot

**Review by kitbellew — COMMENTED** [review]
just some cosmetic stuff

File: RELEASING.md (hunk 1 above)
Comment (dos65) [reply RELEASING.md]: thx!  :smile:

**Review by tgodzik — APPROVED** [review]
LGTM!

**Pull request merged**